package.json

```diff
@@ -179,6 +179,11 @@
       "import": "./dist/middleware/jwt/index.js",
       "require": "./dist/cjs/middleware/jwt/index.js"
     },
+    "./timeout": {
+      "types": "./dist/types/middleware/timeout/index.d.ts",
+      "import": "./dist/middleware/timeout/index.js",
+      "require": "./dist/cjs/middleware/timeout/index.js"
+    },
```

Comment:
Could you add a definition in `typesVersions` too?

https://github.com/honojs/hono/blob/main/package.json#L343

Reply:
ok.
https://github.com/honojs/hono/pull/2615/commits/14e516478809dede603b922e58223d08c485ad55